bohra/SnpDetection.py

```diff
@@ -778,14 +779,29 @@ class TestBohra(SetupInputFiles):
 
     def __init__(self,args):
 
-        self.read_path = args.read_path
+        self.read_path = f"{pathlib.Path.cwd() / 'test_data'}"
+        self.isolate_list = ['ERR1102348','ERR1102353','ERR1102355','ERR1102356']
+        self.download_stub = "https://github.com/MDU-PHL/bohra/tree/make_tests_suite/bohra/tests/data/"
```

Comment:
This path will be dead once `make_test_suite` is merged and you delete the branch, should `master` be reliant on existence of this dev branch?

Reply:
https://github.com/MDU-PHL/bohra/blob/98e390a15d246c011974043cf3dfbe8b94027f69/bohra/SnpDetection.py#L784

Reply:
I think you can just use a [permalink](https://docs.github.com/en/repositories/working-with-files/using-files/getting-permanent-links-to-files) instead, which should remain even after deleting `make_test_suite`:

https://github.com/MDU-PHL/bohra/tree/98e390a15d246c011974043cf3dfbe8b94027f69/bohra/tests/data